Library/Formula/dcadec.rb

```diff
@@ -0,0 +1,16 @@
+class Dcadec < Formula
+  homepage "https://github.com/foo86/dcadec"
+  version "7a5479454156c7a2223c7452f2a965389dce5e67"
+  url "https://github.com/foo86/dcadec/archive/7a5479454156c7a2223c7452f2a965389dce5e67.tar.gz"
```

Comment:
Looks like there'll be one available soon: https://github.com/foo86/dcadec/issues/30